### ISSUE TITLE
Add support for Airflow 2.0 as pipeline runtime

### ISSUE THREAD

**Issue by ptitzler** [elyra-ai/elyra#2392]
**Is your feature request related to a problem? Please describe.**

Support for Airflow 1.10 is now limited, as [EOL was reached June 2021](https://airflow.apache.org/docs/apache-airflow/stable/installation/supported-versions.html). 

**Describe the solution you'd like**
Add support for Apache Airflow 2.0

**Comment by jhgoebbert** [comment]
Are there any news on Apache Airflow 2.x support?

I read that it "might" work but it is not tested [in the Elyra documentation](https://elyra.readthedocs.io/en/latest/recipes/configure-airflow-as-a-runtime.html#prerequisites).

**Comment by ptitzler** [comment]
@lresende

**Comment by lresende** [comment]
I was able to get Airflow "runtime specific components" working well, but was not able to try the generic components.

NOT A CONTRIBUTION

**Comment by jhgoebbert** [comment]
> I was able to get Airflow "runtime specific components" working well, but was not able to try the generic components.

That are good news. So there is a high chance that Apache Airflow 2.x actually might work out-of-the-box.  
Or did you have to patch Elyra for this?

**Comment by shalberd** [comment]
Yes, it seems the generic Airflow operators are working (Airflow Package Catalog Connector), while the Airflow Provider Package Catalog Connectors are not, especially with providers written for Airflow 2.x. E.g. in its version for Airflow 2.x, the https://pypi.org/project/apache-airflow-providers-amazon/#files is not working, too.

I was interested in using the DataRobot Provider for Airflow 2.x in the Elyra context, but this, too, is not yet possible.

https://github.com/datarobot/airflow-provider-datarobot/issues/8
